packages/firefuel/test/src/firefuel_test.dart

```diff
@@ -16,4 +16,38 @@ void main() {
       expect(Firefuel.firestore, isA<FirebaseFirestore>());
     });
   });
+
+  group('#env', () {
+    test('should return an empty string when not initialized', () {
+      expect(Firefuel.env, isEmpty);
+    });
+
+    test('should return an empty string when not provided', () {
+      Firefuel.initialize(FakeFirebaseFirestore());
```

Comment:
move `FakeFirebaseFirestore` into a late variable for this group. The creation of `FirebaseFirestore` isn't relevant to the focus of these tests so it can be moved to a `setUp` block.